README.md

```
@@ -1,7 +1,7 @@
-# SIGIL [Dev]
-**Open-source, offline zero-trust encryption utility with multi-layer chaining**
+# SIGIL 
```

markdownlint-cli2 (0.20.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)